src/main/ui/actions/chat.cljs

```diff
@@ -143,16 +146,15 @@
                                                  child-uid
                                                  (str "**User:** " cstr)
                                                  m-uid
-                                                 order))))))
-
-       (<p! (create-new-block c-uid "first" "" ()))
-       (<p! (js/Promise. (fn [_]
+                                                 order)))))))
+     (<p! (create-new-block c-uid "first" "" ()))
```

Comment:
This is regression from previous work, not sure when this was introduced. If the user added more than 1 chat message due to this bug we only extracted the first message instead of all of them.